toolchain/parse/node_kind.def

```diff
@@ -681,6 +681,35 @@ CARBON_PARSE_NODE_KIND_BRACKET(InterfaceDecl, InterfaceIntroducer,
                                CARBON_TOKEN(Semi)
                                    CARBON_IF_ERROR(CARBON_TOKEN(Interface)))
 
+// `impl ... as`:
+//     ImplIntroducer
+//     _repeated_ _external_: AccessModifierKeyword or DeclModifierKeyword
+//         _external_: ImplicitParamList
+//       ImplForall
+//     _optional_
+//     _optional_ _external_: expression
+//     ImplAs
+//     _external_: expression
```

Comment:
Isn't the expression kind of an argument to `as`? Should it be a child instead of a sibling?

Reply:
Okay, just wanted to make sure you're considered it.